<main
  class="c-article-main-column u-float-left js-main-column"
  data-track-component="article body"
>
  <div
    class="c-context-bar u-hide"
    data-component="context-bar"
    aria-hidden="true"
  >
    <div class="c-context-bar__container u-container">
      <div class="c-context-bar__title">
        <!-- Inference and analysis of cell-cell communication using CellChat -->
        scATAC Landscape 结题报告
      </div>

      <div class="c-pdf-download u-clear-both">
        <a
          href="./result_file/report_cn.pdf"
          class="c-pdf-download__link"
          data-article-pdf="true"
          data-readcube-pdf-url="true"
          data-test="download-pdf"
          data-draft-ignore="true"
          data-track="click"
          data-track-action="download pdf"
          data-track-label="link"
          data-track-external=""
        >
          <span>Download PDF</span>
          <svg
            aria-hidden="true"
            focusable="false"
            width="16"
            height="16"
            class="u-icon"
          >
            <use xlink:href="#global-icon-download"></use>
          </svg>
        </a>
      </div>
    </div>
  </div>

  <article lang="en">
    <div class="c-article-header">
      <header>
        <ul class="c-article-identifiers" data-test="article-identifier">
          <li
            class="c-article-identifiers__item"
            data-test="article-category"
          >
            Report
          </li>

          <li class="c-article-identifiers__item">
            <span
              class="c-article-identifiers__open"
              data-test="open-access"
              >scATAC Landscape</span
            >
          </li>

          <li class="c-article-identifiers__item">
            <a
              href="#article-info"
              data-track="click"
              data-track-action="publication date"
              data-track-label="link"
              >Published:
              <time datetime="2021-10-09">{{ date }}</time></a
            >
          </li>
        </ul>

        <h1
          class="c-article-title"
          data-test="article-title"
          data-article-title=""
        >
        {{ summary.title }} scATAC-seq结题报告
        </h1>

      </header>
    </div>

    <div class="c-article-body">
      <section
        aria-labelledby="Abs1"
        data-title="Abstract"
        data-gtm-vis-recent-on-screen-10482319_393="475"
        data-gtm-vis-first-on-screen-10482319_393="475"
        data-gtm-vis-total-visible-time-10482319_393="10000"
        data-gtm-vis-recent-on-screen-10482319_401="475"
        data-gtm-vis-first-on-screen-10482319_401="475"
        data-gtm-vis-total-visible-time-10482319_401="10000"
        data-gtm-vis-has-fired-10482319_393="1"
        data-gtm-vis-has-fired-10482319_401="1"
        lang="en"
      >
        <div class="c-article-section" id="Abs1-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="Abs1"
          >
          结果概述
          </h2>
          <div class="c-article-section__content" id="Abs1-content">
            <p>
              本项目中分析的物种为：{{ summary.Species }}；分析时样本总数为：{{ summary.sample_num }}；<br>
              所有样本过滤前总细胞数为：{{ summary.raw_cell }} cells，过滤后总细胞数为：{{ summary.cell_filter }} cells；<br>
              过滤前平均每个样本的细胞数目为：{{ summary.raw_cell_per_sample }} cells，过滤后为：{{ summary.cell_filter_per_sample }} cells；<br>
              详细结果见细胞信息统计总表。<br>
            </p>
            <div
              class="c-article-table"
              data-test="inline-table"
              data-container-section="table"
              id="table-1"
            >
              <figure>
                <figcaption class="c-article-table__figcaption">
                  <b id="Tab1" data-test="table-caption"
                    >表1 细胞信息统计总表</b
                  >
                </figcaption>
                <div class="c-article-table-container">

                  <div class="c-article-table-border c-table-scroll-wrapper">
                    <div
                      class="
                        c-table-scroll-wrapper__content
                        c-table-scroll-wrapper__fade--transparent
                      "
                      data-component-scroll-wrapper=""
                    >
                      <table class="data last-table">
                        <thead class="c-article-table-head">
                          <tr>
                            {%for col in tables.table1.header %}
                            <th class="u-text-left">{{ col }}</th>
                            {%endfor%}
                          </tr>
                        </thead>
                        <tbody>
                          {% for line in tables.table1.data %}
                          <tr>
                            {% for info in line %}
                            <td class="u-text-left">{{ info }}</td>
                            {% endfor %}
                          </tr>
                          {% endfor %}
                        </tbody>
                      </table>
                    </div>
                  </div>

                  
                  <div class="c-article-section__figure-description" data-test="bottom-caption" id="table-desc">
                    <p>
                      表1 注： <br>Sample ID：分析时使用的样本名；<br>
                      Estimated number of cells：预估的细胞数；<br>
                      Confidently mapped read pairs：有效比对的Reads比例；<br>
                      Mean raw read pairs per cell：平均每个细胞的测序的Reads数；<br>
                      Median high-quality fragments per cell：细胞高质量的片段数的中位数；<br>
                      Fraction of high-quality fragments overlapping TSS：TSS区域高质量片段数比例；<br>
                      Fragments flanking a single nucleosome：单个核小体长度的片段比例；<br>
                      TSS (transcription start site) enrichment score：TSS富集分数，样本一般大于4。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="table-link"
                    data-track="click"
                    data-track-action="view table"
                    data-track-label="button"
                    rel="nofollow"
                    href="./result_file"
                    aria-label="Full size table 1"
                    ><span>Full size table</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xmlns:xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use>
                      </svg></a>
                </div>
              </figure>
            </div>
          </div>
        </div>
      </section>
      <noscript> </noscript>

      <div class="c-pdf-button__container">
        <div class="c-pdf-download u-clear-both">
          <a
            href="./result_file/report_cn.pdf"
            class="c-pdf-download__link"
            data-article-pdf="true"
            data-readcube-pdf-url="true"
            data-test="download-pdf"
            data-draft-ignore="true"
            data-track="click"
            data-track-action="download pdf"
            data-track-label="link"
            data-track-external=""
          >
            <span>Download PDF</span>
            <svg
              aria-hidden="true"
              focusable="false"
              width="16"
              height="16"
              class="u-icon"
            >
              <use xlink:href="#global-icon-download"></use>
            </svg>
          </a>
        </div>
      </div>

      {% if check_sec.cell_filter == 1 %}
      <section data-title="细胞过滤">
        <div class="c-article-section" id="Sec1-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="Sec1"
          >
          细胞过滤
          </h2>
          <div class="c-article-section__content" id="Sec1-content">
            <p>
              原始数据中存在一些低质量的数据信息，会影响生信分析的结果可靠性。因此，在进行下游分析前需要对细胞数据进行过滤，主要指标包括片段数、TSS富集分数和双胞数等。
            </p>
            <p>
              片段即 fragment ，在ATAC-seq实验中，表示的是介于两个转座酶插入位点之间的的DNA片段，由于在建库过程中会进行片段长度筛选，只会保留较短的DNA片段，一般是1 kb以下。片段数反映了Tn5转座酶在基因组的插入情况，是评价scATAC-seq实验成功与否的重要指标。一般来说，片段数在1000以下的细胞其携带的信息过少，不利于进行下游分析，应予以舍弃。 
            </p>
            <p>
              转录活跃的基因其转录起始位点区域是开放的，有利于Tn5转座酶插入。TSS富集分数，反应的就是TSS区域片段数所占的比例，可以用来评估噪声情况，如果TSS富集分数太低，说明Tn5转座的效果不理想，获得的文库中存在较大比例的基因组背景。
            </p>
            <p>
              双胞或者多胞会对下游的数据分析产生影响，ArhcR
              <sup>
                <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Granja J M, Corces M R, Pierce S E, et al. ArchR is a scalable software package for integrative single-cell chromatin accessibility analysis[J]. Nature genetics, 2021, 53(3): 403-411." href="./report.html#ref-CR1" id="ref-link-section-d38771043e817">1</a>
              </sup>
              可以通过比较单个barcode与来自两个barcode的模拟混合数据的相似性，来评估该barcode是双胞的可能性。具体指标如下：
            </p>
            <p>
              (1)  片段数：2000 ~ 100, 000；<br>
              (2)  TSS富集分数：≥ 6；<br>
              (3)  双胞分数：默认去除双胞分数最高的5%的细胞。
            </p>

            <div
              class="c-article-table"
              data-test="inline-table"
              data-container-section="table"
              id="table-2"
            >
              <figure>
                <figcaption class="c-article-table__figcaption">
                  <b id="Tab2" data-test="table-caption"
                    >表2 过滤后细胞信息统计表</b
                  >
                </figcaption>
                <div class="c-article-table-container">

                  <div class="c-article-table-border c-table-scroll-wrapper">
                    <div
                      class="
                        c-table-scroll-wrapper__content
                        c-table-scroll-wrapper__fade--transparent
                      "
                      data-component-scroll-wrapper=""
                    >
                      <table class="data last-table">
                        <thead class="c-article-table-head">
                          <tr>
                            {%for col in tables.table2.header %}
                            <th class="u-text-left">{{ col }}</th>
                            {%endfor%}
                          </tr>
                        </thead>
                        <tbody>
                          {% for line in tables.table2.data %}
                          <tr>
                            {% for info in line %}
                            <td class="u-text-left">{{ info }}</td>
                            {% endfor %}
                          </tr>
                          {% endfor %}
                        </tbody>
                      </table>
                    </div>
                  </div>
                  
                  <div class="c-article-section__figure-description" data-test="bottom-caption" id="table-desc">
                    <p>
                      表1 注： <br>Sample：分析时使用的样本名称；<br>
                      Cells：过滤后的细胞数；<br>
                      Median Frags：过滤后细胞内片段数的中位数；<br>
                      median TSSEnrichment：过滤后细胞内TSS富集分数的中位数；<br>
                      median DoubletEnrichment：过滤后细胞内是双胞富集分数的中位数；<br>
                      fragments flanking a single nucleosome：过滤后单个核小体长度的片段比例。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="table-link"
                    data-track="click"
                    data-track-action="view table"
                    data-track-label="button"
                    rel="nofollow"
                    href="./result_file"
                    aria-label="Full size table 2"
                    ><span>Full size table</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xmlns:xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use>
                      </svg></a>
                </div>
              </figure>
            </div>

            <p>
              对数据进行过滤后，可以看到细胞的TSS富集分数和片段数等指标均得到显著提升（Fig. 1a-d, f）。由于Tn5转座子的特性，片段长度的分布会与核小体长度的倍数呈现周期性的变化，这也是评价ATAC数据质量的参考依据，通常在单个核小体和两个核小体长度的片段比例会比较高（Fig. 1e）。            
            </p>

            <div class="
                c-article-section__figure
                js-c-reading-companion-figures-item
              " data-test="figure" data-container-section="figure" id="figure-1" data-title="克隆型分布" data-gtm-vis-first-on-screen-10482319_399="746637" data-gtm-vis-total-visible-time-10482319_399="10000" data-gtm-vis-recent-on-screen-10482319_399="4647217" data-gtm-vis-has-fired-10482319_399="1">
              <figure>
                <figcaption>
                  <b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: 数据质控</b>
                </figcaption>
                <div class="c-article-section__figure-content">

                  <div class="c-article-section__figure-item">
                    <a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="./full_size_page/Fig1.html" rel="nofollow"><picture><source type="image/webp" srcset="
                      show_img/{{ figs_name.fig1 }} 1x
                          ">
                        <img aria-describedby="Fig0" src="show_img/{{ figs_name.fig1 }}" alt="figure0" loading="lazy" width="685" height="519"></picture></a>
                  </div>

                  <div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc">
                    <p>
                      <b>a</b> 过滤前样本的片段数分布。
                      <b>b</b> 过滤前样本的TSS富集分数的分布。 
                      <b>c</b> 过滤后样本的片段数分布。
                      <b>d</b> 过滤后样本的TSS富集分数的分布。
                      <b>e</b> 过滤后样本长度的分布。
                      <b>f</b> 过滤后样本的片段数和TSS富集分数的分布密度图，过滤前分样本的片段数和TSS富集分数的分布密度图见<a href="../Result/Step2/sample">密度图文件夹</a>。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="./full_size_page/Fig1.html" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon">
                      <use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a>
                </div>
              </figure>
            </div>
          </div>
        </div>
      </section>
      {% endif %}

      {% if check_sec.sample_merge == 1 %}
      <section data-title="数据整合">
        <div class="c-article-section" id="Sec11-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="Sec11"
          >
          数据整合
          </h2>
          <div class="c-article-section__content" id="Sec11-content">
            <p>
              对多批次、多样本的数据分析时，scATAC-seq数据一般会显示出较强的批次效应。批次效应会干扰下游的分析，影响挖掘生物学意义的差异，这里我们使用harmony对批次效应进行校正。
            </p>
            <div
              class="
                c-article-section__figure
                js-c-reading-companion-figures-item
              "
              data-test="figure"
              data-container-section="figure"
              id="figure-2"
              data-title="Overview of CellChat."
            >
              <figure>
                <figcaption>
                  <b
                    id="Fig2"
                    class="c-article-section__figure-caption"
                    data-test="figure-caption-text"
                    >Fig. 2: 数据整合</b
                  >
                </figcaption>

                <div class="c-article-section__figure-content">
                  <div class="c-article-section__figure-item">
                    <a class="c-article-section__figure-link" data-test="img-link" data-track="click" 
                    data-track-label="image" data-track-action="view figure" href="./full_size_page/Fig2.html" rel="nofollow">
                      <picture><source type="image/webp" srcset="
                      show_img/{{ figs_name.fig2 }} 1x
                          ">
                        <img aria-describedby="Fig2" src="show_img/{{ figs_name.fig1 }}" alt="figure0" loading="lazy" width="685" height="519"></picture></a>
                  </div>

                  <div
                    class="c-article-section__figure-description"
                    data-test="bottom-caption"
                    id="figure-2-desc"
                  >
                    <p>
                      <b>a</b> 使用harmony校正批次效应后，按样本显示的UMAP图。 
                      <b>b</b> 使用harmony校正批次效应后，按批次显示的UMAP图。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="article-link"
                    data-track="click"
                    data-track-label="button"
                    data-track-action="view figure"
                    href="./full_size_page/Fig2.html"
                    data-track-dest="link:Figure1 Full size image"
                    aria-label="Full size image figure 1"
                    rel="nofollow"
                    ><span>Full size image</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use></svg></a>
                </div>
              </figure>
            </div>
          </div>
        </div>
      </section>
      {% endif %}

      {% if check_sec.cell_cluster_marker == 1 %}
      <section data-title="类特征分析">
        <div class="c-article-section" id="Sec12-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="Sec12"
          >
          类特征分析
          </h2>
          <div class="c-article-section__content" id="Sec12-content">
            <!-- <h3 class="c-article__sub-heading" id="Sec13">
              Database construction for ligand-receptor interactions
            </h3> -->
            <p>
              相似的细胞通常呈现相近的染色质开放状态。通过聚类分析，将相似的细胞归为一群，有助于发现一类细胞间普遍存在的染色质开放状态的规律。ArchR使用源自Seurat
              <sup>
                <a data-track="click" data-track-action="reference anchor" 
                data-track-label="link" data-test="citation-ref" aria-label="Reference 2" 
                title="Hao Y, Hao S, Andersen-Nissen E, et al. Integrated analysis of multimodal single-cell data[J]. Cell, 2021, 184(13): 3573-3587." 
                href="./report.html#ref-CR2" id="ref-link-section-d38771043e817">2</a>
              </sup>
              的聚类方法，这里我们使用resolution = {{ summary.resolution }} 进行聚类（Fig. 3a）。
            </p>
            <p>
              ArchR通过对基因转录起始位点附近5kb区域的片段数按离TSS的距离进行加权分析，获得细胞中每一个基因的表达分数（gene score）。通过对类群中基因表达分数的差异分析，可以获得每个类群的特征基因（表3），用于对类群的注释。
            </p>

            <div
              class="c-article-table"
              data-test="inline-table"
              data-container-section="table"
              id="table-3"
            >
              <figure>
                <figcaption class="c-article-table__figcaption">
                  <b id="Tab3" data-test="table-caption"
                    >表3 特征基因列表</b
                  >
                </figcaption>
                <div class="c-article-table-container">

                  <div class="c-article-table-border c-table-scroll-wrapper">
                    <div
                      class="
                        c-table-scroll-wrapper__content
                        c-table-scroll-wrapper__fade--transparent
                      "
                      data-component-scroll-wrapper=""
                    >
                      <table class="data last-table">
                        <thead class="c-article-table-head">
                          <tr>
                            {%for col in tables.table3.header %}
                            <th class="u-text-left">{{ col }}</th>
                            {%endfor%}
                          </tr>
                        </thead>
                        <tbody>
                          {% for line in tables.table3.data %}
                          <tr>
                            {% for info in line %}
                            <td class="u-text-left">{{ info }}</td>
                            {% endfor %}
                          </tr>
                          {% endfor %}
                        </tbody>
                      </table>
                    </div>
                  </div>
                  
                  <div class="c-article-section__figure-description" data-test="bottom-caption" id="table-desc">
                    <p>
                      表3 注： <br>group：类群名简称；<br>
                      group_name：类群名；<br>
                      seqnames：特征基因所在染色体；<br>
                      start：特征基因的转录起始位置；<br>
                      end：特征基因的转录中止位置；<br>
                      strand：特征基因转录方向；<br>
                      name：特征基因的名称；<br>
                      idx：特征基因在分析时的序列号；<br>
                      Log2FC：特征基因在该类群中相比其余类群的基因表达分数比值的对数；<br>
                      FDR：假阳性率；<br>
                      MeanDiff：特征基因在该类群中相比其余类群的基因表达分数的差值。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="table-link"
                    data-track="click"
                    data-track-action="view table"
                    data-track-label="button"
                    rel="nofollow"
                    href="./result_file"
                    aria-label="Full size table 2"
                    ><span>Full size table</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xmlns:xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use>
                      </svg></a>
                </div>
              </figure>
            </div>

            <p>
              为了更好地反映每一类群特征基因的分布情况，我们同时绘制了特征基因的小提琴图、在转录起始位置附近的片段分布峰图和UMAP展示图（Fig. 3b-d）。
            </p>

            <div
              class="
                c-article-section__figure
                js-c-reading-companion-figures-item
              "
              data-test="figure"
              data-container-section="figure"
              id="figure-3"
              data-title="Overview of CellChat."
            >
              <figure>
                <figcaption>
                  <b
                    id="Fig3"
                    class="c-article-section__figure-caption"
                    data-test="figure-caption-text"
                    >Fig. 3: 聚类分析</b
                  >
                </figcaption>
                <div class="c-article-section__figure-content">

                  <div class="c-article-section__figure-item">
                    <a class="c-article-section__figure-link" data-test="img-link" data-track="click" 
                    data-track-label="image" data-track-action="view figure" href="./full_size_page/Fig3.html" rel="nofollow">
                      <picture><source type="image/webp" srcset="
                      show_img/{{ figs_name.fig3 }} 1x
                          ">
                        <img aria-describedby="Fig3" src="show_img/{{ figs_name.fig3 }}" alt="figure3" loading="lazy" width="685" height="519"></picture></a>
                  </div>

                  <div
                    class="c-article-section__figure-description"
                    data-test="bottom-caption"
                    id="figure-3-desc"
                  >
                    <p>
                      <b>a</b> 细胞类群的UMAP分布图。
                      <b>b</b> 特征基因的小提琴图，默认展示每类的top1的marker。若类群过多，则只展示前10群的marker。
                      <b>c</b> 特征基因的片段分布峰图，以选定基因的TSS为中心，展示向左右两侧各5 kb的区域，片段峰图可以清晰地显示该基因TSS附近的染色质开放状态。
                               仅展示一个基因，其余见<a href="../Result/Step3/Cluster/res{{ summary.resolution }}/Browser">结果文件</a>。
                      <b>d</b> 特征基因gene score的UMAP展示，仅展示每类的top1基因，如果类群多余10个，仅展示10个，更多结果见<a href="../Result/Step3/Cluster/res{{ summary.resolution }}/res{{ summary.resolution }}-UMAP-Marker-Genes-Imputation.pdf">UMAP pdf图</a>。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="article-link"
                    data-track="click"
                    data-track-label="button"
                    data-track-action="view figure"
                    href="./full_size_page/Fig3.html"
                    data-track-dest="link:Figure1 Full size image"
                    aria-label="Full size image figure 1"
                    rel="nofollow"
                    ><span>Full size image</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use></svg></a>
                </div>
              </figure>
            </div>
            <p>
              为了对细胞类群进行更好地刻画，我们绘制了制定类群的差异基因火山图（Fig. 4a），可以显示该群与剩余细胞相比，启动子区域显著更开放和不开放的基因。同时，提供了不同类群的gene score值的差异化热图（Fig. 4b）。
            </p>
          </div>
        </div>
      </section>
      {% endif %}

      {% if check_sec.go_kegg == 1 %}
      <section data-title="基因富集分析">
        <div class="c-article-section" id="code-availability-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="富集分析"
          >
          基因富集分析
          </h2>
          <div
            class="c-article-section__content"
            id="code-availability-content"
          >
          <p>
            GO数据库是GO组织（Gene Ontology Consortium）于2000年构建的一个结构化的标准生物学注释系统，旨在建立基因及其产物知识的标准词汇体系，适用于各个物种。GO注释系统是一个有向无环图，包含三个主要分支，即：生物学过程（Biological Process），分子功能（Molecular Function）和细胞组分（Cellular Component）。对每个cluster的差异基因集，采用ClusterProfiler
            <sup>
              <a data-track="click" data-track-action="reference anchor" 
              data-track-label="link" data-test="citation-ref" aria-label="Reference 3" 
              title="Yu G, Wang L G, Han Y, et al. clusterProfiler: an R package for comparing biological themes among gene clusters[J]. Omics: a journal of integrative biology, 2012, 16(5): 284-287." 
              href="./report.html#ref-CR3" id="ref-link-section-d38771043e817">3</a>
            </sup>
            对基因分别进行生物学过程，分子功能和细胞组分的富集分析。富集分析采用超几何检验方法来寻找与整个基因组背景相比显著富集的GO条目。我们用气泡图展现了所有cluster top的GO term（仅包含BP）（Fig. 4c）。全部的富集结果详见 表4 GO富集分析结果表。
          </p>
          <div
              class="
                c-article-section__figure
                js-c-reading-companion-figures-item
              "
              data-test="figure"
              data-container-section="figure"
              id="figure-4"
              data-title="Overview of CellChat."
            >
              <figure>
                <figcaption>
                  <b
                    id="Fig4"
                    class="c-article-section__figure-caption"
                    data-test="figure-caption-text"
                    >Fig. 4: 基因富集分析</b
                  >
                </figcaption>
                <div class="c-article-section__figure-content">

                  <div class="c-article-section__figure-item">
                    <a class="c-article-section__figure-link" data-test="img-link" data-track="click" 
                    data-track-label="image" data-track-action="view figure" href="./full_size_page/Fig4.html" rel="nofollow">
                      <picture><source type="image/webp" srcset="
                      show_img/{{ figs_name.fig4 }} 1x
                          ">
                        <img aria-describedby="Fig4" src="show_img/{{ figs_name.fig4 }}" alt="figure4" loading="lazy" width="685" height="519"></picture></a>
                  </div>

                  <div
                    class="c-article-section__figure-description"
                    data-test="bottom-caption"
                    id="figure-4-desc"
                  >
                    <p>
                      <b>a</b> 特定类群的特征基因火山图。蓝色为显著下调基因（|Fold Change| ≥ 1.5 & FDR ≤ 0.1），红色为显著上调基因，灰色为没有差异的基因。每个cluster的火山图见<a href="../Result/Step7/res{{ summary.resolution }}/enrichment">差异基因文件夹</a>。
                      <b>b</b> 差异基因热图，仅标示出每群top1的基因。
                      <b>c</b> GO富集分析的气泡图，主要侧重Biological Process这个category。气泡图显示不同cluster top富集的GO Term。点越大，富集到term中的基因越多，颜色深浅表示校正后的P-value。
                      <b>d</b> KEGG富集分析的气泡图，展示了不同cluster top富集的KEGG pathways。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="article-link"
                    data-track="click"
                    data-track-label="button"
                    data-track-action="view figure"
                    href="./full_size_page/Fig4.html"
                    data-track-dest="link:Figure1 Full size image"
                    aria-label="Full size image figure 1"
                    rel="nofollow"
                    ><span>Full size image</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use></svg></a>
                </div>
              </figure>
            </div>

            <div
              class="c-article-table"
              data-test="inline-table"
              data-container-section="table"
              id="table-4"
            >
              <figure>
                <figcaption class="c-article-table__figcaption">
                  <b id="Tab4" data-test="table-caption"
                    >表4 富集分析结果表</b
                  >
                </figcaption>
                <div class="c-article-table-container">

                  <div class="c-article-table-border c-table-scroll-wrapper">
                    <div
                      class="
                        c-table-scroll-wrapper__content
                        c-table-scroll-wrapper__fade--transparent
                      "
                      data-component-scroll-wrapper=""
                    >
                      <table class="data last-table">
                        <thead class="c-article-table-head">
                          <tr>
                            {%for col in tables.table4.header %}
                            <th class="u-text-left">{{ col }}</th>
                            {%endfor%}
                          </tr>
                        </thead>
                        <tbody>
                          {% for line in tables.table4.data %}
                          <tr>
                            {% for info in line %}
                            <td class="u-text-left">{{ info }}</td>
                            {% endfor %}
                          </tr>
                          {% endfor %}
                        </tbody>
                      </table>
                    </div>
                  </div>
                  
                  <div class="c-article-section__figure-description" data-test="bottom-caption" id="table-desc">
                    <p>
                      表4 注： <br>ID ：GO term；<br>
                      Description ：通路（term）的描述信息；<br>
                      GeneRatio ：富集到该通路（term）的差异基因数/所有的差异基因数目；<br>
                      BgRatio：所有背景基因中富集到该通路（term）中的基因数/所有背景基因集；<br>
                      enrich_factor：GeneRation/BgRatio；<br>
                      pvalue：富集的pvalue；<br>
                      qvalue：富集的qvalue；<br>
                      geneID：富集到该通路（term）的差异基因的symbol；<br>
                      gene_number：富集到该通路（term）的差异基因数。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="table-link"
                    data-track="click"
                    data-track-action="view table"
                    data-track-label="button"
                    rel="nofollow"
                    href="./result_file"
                    aria-label="Full size table 2"
                    ><span>Full size table</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xmlns:xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use>
                      </svg></a>
                </div>
              </figure>
            </div>
            <p>
              KEGG(Kyoto Encyclopedia of Genes and Genomes)是一个整合了基因组、化学和系统功能信息的数据库。基于KEGG数据库，pathway 富集分析用超几何检验找出与整个基因组背景相比，在差异基因中显著富集的 pathway。气泡图展示了所有cluster top的KEGG pathway，并用颜色表示了pathway的富集程度（Fig. 4d）。全部结果详见 表5 KEGG富集分析结果表。
            </p>

            <div
              class="c-article-table"
              data-test="inline-table"
              data-container-section="table"
              id="table-5"
            >
              <figure>
                <figcaption class="c-article-table__figcaption">
                  <b id="Tab5 data-test="table-caption"
                    >表5 KEGG富集分析结果表</b
                  >
                </figcaption>
                <div class="c-article-table-container">

                  <div class="c-article-table-border c-table-scroll-wrapper">
                    <div
                      class="
                        c-table-scroll-wrapper__content
                        c-table-scroll-wrapper__fade--transparent
                      "
                      data-component-scroll-wrapper=""
                    >
                      <table class="data last-table">
                        <thead class="c-article-table-head">
                          <tr>
                            {%for col in tables.table5.header %}
                            <th class="u-text-left">{{ col }}</th>
                            {%endfor%}
                          </tr>
                        </thead>
                        <tbody>
                          {% for line in tables.table5.data %}
                          <tr>
                            {% for info in line %}
                            <td class="u-text-left">{{ info }}</td>
                            {% endfor %}
                          </tr>
                          {% endfor %}
                        </tbody>
                      </table>
                    </div>
                  </div>
                  
                  <div class="c-article-section__figure-description" data-test="bottom-caption" id="table-desc">
                    <p>
                      
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="table-link"
                    data-track="click"
                    data-track-action="view table"
                    data-track-label="button"
                    rel="nofollow"
                    href="./result_file"
                    aria-label="Full size table 2"
                    ><span>Full size table</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xmlns:xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use>
                      </svg></a>
                </div>
              </figure>
            </div>

          </div>
        </div>
      </section>
      {% endif %}

      {% if check_sec.pseudotime == 1 %}
      <section data-title="Motif富集分析">
        <div class="c-article-section" id="article-comments-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="Motif富集分析"
          >
          Motif富集分析
          </h2>
          <div
            class="c-article-section__content"
            id="article-comments-content"
          >
            <p>
              Peak Calling是ATAC-seq常用的分析方法，可以更为准确地获知片段的峰所在位置。为了提高peaks的可靠性，ArchR将同属一个类群的细胞合并为数个伪bulk数据，利用MACS2
              <sup>
                <a data-track="click" data-track-action="reference anchor" 
                data-track-label="link" data-test="citation-ref" aria-label="Reference 4" 
                title="Zhang Y, Liu T, Meyer C A, et al. Model-based analysis of ChIP-Seq (MACS)[J]. Genome biology, 2008, 9(9): 1-9." 
                href="./report.html#ref-CR4" id="ref-link-section-d38771043e817">4</a>
              </sup>
              对伪bulk数据进行peak calling，并进行假设检验，仅保留类群中保守性较好的峰进行下游分析。利用这些类群保守的峰进行motif富集分析，可以获得类群活跃的转录因子，以便于更为全面地对类群特征进行刻画。
            </p>
            <p>
              我们用专为单细胞数据开发的chromVAR
              <sup>
                <a data-track="click" data-track-action="reference anchor" 
                data-track-label="link" data-test="citation-ref" aria-label="Reference 5" 
                title="Schep A N, Wu B, Buenrostro J D, et al. chromVAR: inferring transcription-factor-associated accessibility from single-cell epigenomic data[J]. Nature methods, 2017, 14(10): 975-978." 
                href="./report.html#ref-CR5" id="ref-link-section-d38771043e817">5</a>
              </sup>
              进行motif分析，得到了在所有类群中motif的差异排序（Fig. 5a）。对类群motif富集分数进行z-score标准化，然后按照z-score值排序，可以获得类群特异的motif（Fig. 5b-e）。
            </p>
            <div
              class="
                c-article-section__figure
                js-c-reading-companion-figures-item
              "
              data-test="figure"
              data-container-section="figure"
              id="figure-5"
              data-title="Overview of CellChat."
            >
              <figure>
                <figcaption>
                  <b
                    id="Fig5"
                    class="c-article-section__figure-caption"
                    data-test="figure-caption-text"
                    >Fig. 5: Motif富集分析</b
                  >
                </figcaption>
                <div class="c-article-section__figure-content">

                  <div class="c-article-section__figure-item">
                    <a class="c-article-section__figure-link" data-test="img-link" data-track="click" 
                    data-track-label="image" data-track-action="view figure" href="./full_size_page/Fig5.html" rel="nofollow">
                      <picture><source type="image/webp" srcset="
                      show_img/{{ figs_name.fig5 }} 1x
                          ">
                        <img aria-describedby="Fig5" src="show_img/{{ figs_name.fig5 }}" alt="figure0" loading="lazy" width="685" height="519"></picture></a>
                  </div>

                  <div
                    class="c-article-section__figure-description"
                    data-test="bottom-caption"
                    id="figure-5-desc"
                  >
                    <p>
                      <b>a</b> 所有类群中motif的变异度排序。纵轴为motif的deviation值，deviation值越大表明该motif在类群间的差异越大，横轴为按照deviation值的由大到小的排序结果。
                      <b>b</b> 类群特异的motif的富集分数热图。蓝色越深，表明motif在该群的特征越明显。
                      <b>c</b> 类群特异的motif的富集分析的z-score在UMAP上的展示，仅展示每类群的富集分数top1的motif，若类群大于6，则只展示前6群。
                      <b>d</b> 每类群的motif富集分数排序，仅展示C1群的结果，更多结果见<a href="../Result/Step6/Cluster">富集分数曲线图</a>。
                      <b>e</b> 类群特异motif的z-score的分布图。横轴为z-score值，曲线图表示该类群中细胞的z-score的分布密度。峰对应的z-score越大，表明motif在该群的特征越显著。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="article-link"
                    data-track="click"
                    data-track-label="button"
                    data-track-action="view figure"
                    href="./full_size_page/Fig5.html"
                    data-track-dest="link:Figure1 Full size image"
                    aria-label="Full size image figure 1"
                    rel="nofollow"
                    ><span>Full size image</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use></svg></a>
                </div>
              </figure>
            </div>
          </div>
        </div>
      </section>
      {% endif %}




      

      <!-- <section data-title="模板">
        <div class="c-article-section" id="article-comments-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="模板"
          >
            模板
          </h2>
          <div
            class="c-article-section__content"
            id="article-comments-content"
          >
            <p>
              内容
            </p>
          </div>
        </div>
      </section> -->



      <div id="MagazineFulltextArticleBodySuffix">
        <section aria-labelledby="Bib1" data-title="References">
          <div class="c-article-section" id="Bib1-section">
            <h2
              class="
                c-article-section__title
                js-section-title js-c-reading-companion-sections-item
              "
              id="Bib1"
            >
              References
            </h2>
            <div class="c-article-section__content" id="Bib1-content">
              <div data-container-section="references">
                <ol class="c-article-references">
                  <li
                    class="
                      c-article-references__item
                      js-c-reading-companion-references-item
                    "
                  >
                    <span class="c-article-references__counter">1.</span>
                    <p class="c-article-references__text" id="ref-CR1">
                      Granja J M, Corces M R, Pierce S E, et al. ArchR is a scalable software package for integrative single-cell chromatin accessibility analysis[J]. Nature genetics, 2021, 53(3): 403-411.
                    </p>
                  </li>
                  <li
                    class="
                      c-article-references__item
                      js-c-reading-companion-references-item
                    "
                  >
                    <span class="c-article-references__counter">2.</span>
                    <p class="c-article-references__text" id="ref-CR2">
                      Hao Y, Hao S, Andersen-Nissen E, et al. Integrated analysis of multimodal single-cell data[J]. Cell, 2021, 184(13): 3573-3587.
                    </p>
                  </li>
                  <li
                    class="
                      c-article-references__item
                      js-c-reading-companion-references-item
                    "
                  >
                    <span class="c-article-references__counter">3.</span>
                    <p class="c-article-references__text" id="ref-CR3">
                      Yu G, Wang L G, Han Y, et al. clusterProfiler: an R package for comparing biological themes among gene clusters[J]. Omics: a journal of integrative biology, 2012, 16(5): 284-287.
                    </p>
                  </li>
                  <li
                    class="
                      c-article-references__item
                      js-c-reading-companion-references-item
                    "
                  >
                    <span class="c-article-references__counter">4.</span>
                    <p class="c-article-references__text" id="ref-CR4">
                      Zhang Y, Liu T, Meyer C A, et al. Model-based analysis of ChIP-Seq (MACS)[J]. Genome biology, 2008, 9(9): 1-9.
                    </p>
                  </li>
                  <li
                    class="
                      c-article-references__item
                      js-c-reading-companion-references-item
                    "
                  >
                    <span class="c-article-references__counter">5.</span>
                    <p class="c-article-references__text" id="ref-CR5">
                      Schep A N, Wu B, Buenrostro J D, et al. chromVAR: inferring transcription-factor-associated accessibility from single-cell epigenomic data[J]. Nature methods, 2017, 14(10): 975-978.
                    </p>
                  </li>
                  
                  
                </ol>

              </div>
            </div>
          </div>
        </section>
      </div>


      <!-- <span data-recommended="jobs"></span> -->
    </div>
  </article>
</main>

<aside
  class="c-article-extras u-hide-print"
  aria-label="Article navigation"
  data-component-reading-companion=""
  data-container-type="reading-companion"
  data-track-component="reading companion"
>
  <div class="c-pdf-download u-clear-both">
    <a
      href="./result_file/report_cn.pdf"
      class="c-pdf-download__link"
      data-article-pdf="true"
      data-readcube-pdf-url="true"
      data-test="download-pdf"
      data-draft-ignore="true"
      data-track="click"
      data-track-action="download pdf"
      data-track-label="link"
      data-track-external=""
    >
      <span>Download PDF</span>
      <svg
        aria-hidden="true"
        focusable="false"
        width="16"
        height="16"
        class="u-icon"
      >
        <use xlink:href="#global-icon-download"></use>
      </svg>
    </a>
  </div>

  <div class="c-reading-companion">
    <div
      class="c-reading-companion__sticky"
      data-component="reading-companion-sticky"
      data-test="reading-companion-sticky"
      style="top: 40px"
    >
      <div
        class="
          c-reading-companion__panel
          c-reading-companion__sections
          c-reading-companion__panel--active
        "
        id="tabpanel-sections"
        aria-labelledby="tab-sections"
      >
        <div class="u-mt-16" data-component-mpu="">
        </div>
      </div>
      <div
        class="
          c-reading-companion__panel
          c-reading-companion__figures
          c-reading-companion__panel--full-width
        "
        id="tabpanel-figures"
        aria-labelledby="tab-figures"
      >
      </div>
      <div
        class="
          c-reading-companion__panel
          c-reading-companion__references
          c-reading-companion__panel--full-width
        "
        id="tabpanel-references"
        aria-labelledby="tab-references"
      >
     </div>
    </div>
  </div>
</aside>
