<main
  class="c-article-main-column u-float-left js-main-column"
  data-track-component="article body"
>
  <div
    class="c-context-bar u-hide"
    data-component="context-bar"
    aria-hidden="true"
  >
    <div class="c-context-bar__container u-container">
      <div class="c-context-bar__title">
        <!-- Inference and analysis of cell-cell communication using CellChat -->
        Immune Landscape 结题报告
      </div>

      <div class="c-pdf-download u-clear-both">
        <a
          href="./result_file/report_cn.pdf"
          class="c-pdf-download__link"
          data-article-pdf="true"
          data-readcube-pdf-url="true"
          data-test="download-pdf"
          data-draft-ignore="true"
          data-track="click"
          data-track-action="download pdf"
          data-track-label="link"
          data-track-external=""
        >
          <span>Download PDF</span>
          <svg
            aria-hidden="true"
            focusable="false"
            width="16"
            height="16"
            class="u-icon"
          >
            <use xlink:href="#global-icon-download"></use>
          </svg>
        </a>
      </div>
    </div>
  </div>

  <article lang="en">
    <div class="c-article-header">
      <header>
        <ul class="c-article-identifiers" data-test="article-identifier">
          <li
            class="c-article-identifiers__item"
            data-test="article-category"
          >
            Report
          </li>

          <li class="c-article-identifiers__item">
            <span
              class="c-article-identifiers__open"
              data-test="open-access"
              >Immune Landscape</span
            >
          </li>

          <li class="c-article-identifiers__item">
            <a
              href="#article-info"
              data-track="click"
              data-track-action="publication date"
              data-track-label="link"
              >Published:
              <time datetime="2021-10-09">{{ date }}</time></a
            >
          </li>
        </ul>

        <h1
          class="c-article-title"
          data-test="article-title"
          data-article-title=""
        >
        {{ summary.title }} 免疫组库结题报告
        </h1>

      </header>
    </div>

    <div class="c-article-body">
      <section
        aria-labelledby="Abs1"
        data-title="Abstract"
        data-gtm-vis-recent-on-screen-10482319_393="475"
        data-gtm-vis-first-on-screen-10482319_393="475"
        data-gtm-vis-total-visible-time-10482319_393="10000"
        data-gtm-vis-recent-on-screen-10482319_401="475"
        data-gtm-vis-first-on-screen-10482319_401="475"
        data-gtm-vis-total-visible-time-10482319_401="10000"
        data-gtm-vis-has-fired-10482319_393="1"
        data-gtm-vis-has-fired-10482319_401="1"
        lang="en"
      >
        <div class="c-article-section" id="Abs1-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="Abs1"
          >
          结果概述
          </h2>
          <div class="c-article-section__content" id="Abs1-content">
            <p>
              本项目中分析物种为：{{ summary.species }}<br>
              <p>
                分析的TCR样本总数为：{{ summary.TCR.sample_num }}个；<br>
                所有样本过滤前总细胞数为：{{ summary.TCR.raw_cell_num }}个，过滤后带有效V(D)J对的总细胞数为：{{ summary.TCR.filted_VJ_cell_num }}个；<br>
                过滤前平均每个样本的细胞数为：{{ summary.TCR.raw_cell_num_per_sample }}个，过滤后为：{{ summary.TCR.filted_VJ_cell_num_per_sample }}个；<br>
                平均每个样本的V(D)J片段比对率为：{{ summary.TCR.map_vdj_rate }}。详细结果见 表1 细胞信息统计总表。
              </p>
              <p>
                分析的BCR样本总数为：{{ summary.BCR.sample_num }}个；<br>
                所有样本过滤前总细胞数为：{{ summary.BCR.raw_cell_num }}个，过滤后带有效V(D)J对的总细胞数为：{{ summary.BCR.filted_VJ_cell_num }}个；<br>
                过滤前平均每个样本的细胞数为：{{ summary.BCR.raw_cell_num_per_sample }}个，过滤后为：{{ summary.BCR.filted_VJ_cell_num_per_sample }}个；<br>
                平均每个样本的V(D)J片段比对率为：{{ summary.BCR.map_vdj_rate }}。详细结果见 表1 细胞信息统计总表。
              </p>
            </p>
            <div
              class="c-article-table"
              data-test="inline-table"
              data-container-section="table"
              id="table-1"
            >
              <figure>
                <figcaption class="c-article-table__figcaption">
                  <b id="Tab1" data-test="table-caption"
                    >表1 细胞信息统计总表</b
                  >
                </figcaption>
                <div class="c-article-table-container">

                  {% if summary.TCR.has %}
                  TCR
                  <!-- <p>
                    分析的TCR样本总数为：{{ summary.TCR.sample_num }}个；<br>
                    所有样本过滤前总细胞数为：{{ summary.TCR.raw_cell_num }}个，过滤后带有效V(D)J对总细胞数为：{{ summary.TCR.filted_VJ_cell_num }}个；<br>
                    过滤前平均每个样本的细胞数为：{{ summary.TCR.raw_cell_num_per_sample }}个，过滤后为：{{ summary.TCR.filted_VJ_cell_num_per_sample }}个；<br>
                    平均每个样本比对到V(D)J片段的比对率为：{{ summary.TCR.map_vdj_rate }}，详细结果见细胞信息统计总表。
                  </p> -->
                  <div class="c-article-table-border c-table-scroll-wrapper">
                    <div
                      class="
                        c-table-scroll-wrapper__content
                        c-table-scroll-wrapper__fade--transparent
                      "
                      data-component-scroll-wrapper=""
                    >
                      <table class="data last-table">
                        <thead class="c-article-table-head">
                          <tr>
                            {%for col in TCR.{{ table_keys | first }}.header %}
                            <th class="u-text-left">{{ col }}</th>
                            {%endfor%}
                          </tr>
                        </thead>
                        <tbody>
                          {% for line in TCR_summary.data %}
                          <tr>
                            {% for info in line %}
                            <td class="u-text-left">{{ info }}</td>
                            {% endfor %}
                          </tr>
                          {% endfor %}
                        </tbody>
                      </table>
                    </div>
                  </div>
                  {% endif %}
                  {% if summary.BCR.has %}
                  <br>
                  BCR
                  <!-- <p>
                    分析的BCR样本总数为：{{ summary.BCR.sample_num }}个；<br>
                    所有样本过滤前总细胞数为：{{ summary.BCR.raw_cell_num }}个，过滤后带有效V(D)J对总细胞数为：{{ summary.BCR.filted_VJ_cell_num }}个；<br>
                    过滤前平均每个样本的细胞数为：{{ summary.BCR.raw_cell_num_per_sample }}个，过滤后为：{{ summary.BCR.filted_VJ_cell_num_per_sample }}个；<br>
                    平均每个样本比对到V(D)J片段的比对率为：{{ summary.BCR.map_vdj_rate }}，详细结果见细胞信息统计总表。
                  </p> -->
                  <div class="c-article-table-border c-table-scroll-wrapper">
                    <div
                      class="
                        c-table-scroll-wrapper__content
                        c-table-scroll-wrapper__fade--transparent
                      "
                      data-component-scroll-wrapper=""
                    >
                      <table class="data last-table">
                        <thead class="c-article-table-head">
                          <tr>
                            {%for col in BCR_summary.header %}
                            <th class="u-text-left">{{ col }}</th>
                            {%endfor%}
                          </tr>
                        </thead>
                        <tbody>
                          {% for line in BCR_summary.data %}
                          <tr>
                            {% for info in line %}
                            <td class="u-text-left">{{ info }}</td>
                            {% endfor %}
                          </tr>
                          {% endfor %}
                        </tbody>
                      </table>
                    </div>
                  </div>
                  {% endif %}
                  
                  <div class="c-article-section__figure-description" data-test="bottom-caption" id="table-desc">
                    <p>
                      表1 注： <br>SampleID：分析时的样本名；<br>
                      Estimated Number of Cells：检测到的细胞数目，细胞数目取决于文库中T/B细胞比例；<br>
                      Mean Read Pairs per Cell：平均每个细胞检测到的Reads数目，推荐每个细胞的测序量为5000个reads；<br>
                      Number of Read Pairs：样本的总reads数目；<br>
                      Number of Cells With Productive V-J Spanning Pair：带有效V-J对的细胞数目；<br>
                      Reads Mapped to Any V(D)J Gene：比对到V(D)J片段的Reads比例；<br>
                      Reads Mapped to TRA/B, IGH/K/L：比对到相应片段的Reads比例。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="table-link"
                    data-track="click"
                    data-track-action="view table"
                    data-track-label="button"
                    rel="nofollow"
                    href="./result_file"
                    aria-label="Full size table 1"
                    ><span>Full size table</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xmlns:xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use>
                      </svg></a>
                </div>
              </figure>
            </div>
          </div>
        </div>
      </section>
      <noscript> </noscript>

      <div class="c-pdf-button__container">
        <div class="c-pdf-download u-clear-both">
          <a
            href="./result_file/report_cn.pdf"
            class="c-pdf-download__link"
            data-article-pdf="true"
            data-readcube-pdf-url="true"
            data-test="download-pdf"
            data-draft-ignore="true"
            data-track="click"
            data-track-action="download pdf"
            data-track-label="link"
            data-track-external=""
          >
            <span>Download PDF</span>
            <svg
              aria-hidden="true"
              focusable="false"
              width="16"
              height="16"
              class="u-icon"
            >
              <use xlink:href="#global-icon-download"></use>
            </svg>
          </a>
        </div>
      </div>

      {% if check_sec.cell_filter == 1 %}
      <section data-title="克隆型分布">
        <div class="c-article-section" id="Sec1-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="Sec1"
          >
          克隆型分布
          </h2>
          <div class="c-article-section__content" id="Sec1-content">
            <p>
              通常情况下，一个淋巴细胞会同时表达一条重链和一条轻链，这两条链共同确定了这个细胞的克隆型(Clonetype)。
              相比常规免疫组库分析，单细胞免疫组库可以同时获得一个细胞的轻链和重链，因此可以更好地确定细胞的克隆型。
              但在实际的分析过程中，可能会出现一个克隆型只含有一条链或者多条链的情况，在我们的分析中过滤了含有3条以上链的细胞。
            </p>
            <p>
              CDR3区域是T/BCR序列中变化最丰富的部分，两个细胞CDR3序列完全相同被认定为一种克隆型。
              如果两个细胞的一条链完全相同，其中一个细胞未测到另一条链，也视为两个克隆型。在 表2 中包括所有样本的克隆型统计的基本信息。
            </p>

            <div
              class="c-article-table"
              data-test="inline-table"
              data-container-section="table"
              id="table-2"
            >
              <figure>
                <figcaption class="c-article-table__figcaption">
                  <b id="Tab2" data-test="table-caption"
                    >表2 克隆型统计</b
                  >
                </figcaption>
                <div class="c-article-table-container">

                  {% if summary.TCR.has %}
                  TCR
                  <div class="c-article-table-border c-table-scroll-wrapper">
                    <div
                      class="
                        c-table-scroll-wrapper__content
                        c-table-scroll-wrapper__fade--transparent
                      "
                      data-component-scroll-wrapper=""
                    >
                      <table class="data last-table">
                        <thead class="c-article-table-head">
                          <tr>
                            {%for col in TCR_summary.header %}
                            <th class="u-text-left">{{ col }}</th>
                            {%endfor%}
                          </tr>
                        </thead>
                        <tbody>
                          {% for line in TCR_summary.data %}
                          <tr>
                            {% for info in line %}
                            <td class="u-text-left">{{ info }}</td>
                            {% endfor %}
                          </tr>
                          {% endfor %}
                        </tbody>
                      </table>
                    </div>
                  </div>
                  {% endif %}
                  {% if summary.BCR.has %}
                  <br>
                  BCR
                  <div class="c-article-table-border c-table-scroll-wrapper">
                    <div
                      class="
                        c-table-scroll-wrapper__content
                        c-table-scroll-wrapper__fade--transparent
                      "
                      data-component-scroll-wrapper=""
                    >
                      <table class="data last-table">
                        <thead class="c-article-table-head">
                          <tr>
                            {%for col in BCR_summary.header %}
                            <th class="u-text-left">{{ col }}</th>
                            {%endfor%}
                          </tr>
                        </thead>
                        <tbody>
                          {% for line in BCR_summary.data %}
                          <tr>
                            {% for info in line %}
                            <td class="u-text-left">{{ info }}</td>
                            {% endfor %}
                          </tr>
                          {% endfor %}
                        </tbody>
                      </table>
                    </div>
                  </div>
                  {% endif %}
                  
                  <div class="c-article-section__figure-description" data-test="bottom-caption" id="table-desc">
                    <p>
                      表1 注： <br>SampleID：分析时的样本名；<br>
                      Clonetype_id：克隆型名称；<br>
                      Frequency：克隆型大小；<br>
                      Proportion：克隆型占样本的比例；<br>
                      CDR3.aa：克隆型的CDR3序列，如果克隆型为双链，则用分号隔开，前面为轻链后面为重链。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="table-link"
                    data-track="click"
                    data-track-action="view table"
                    data-track-label="button"
                    rel="nofollow"
                    href="./result_file"
                    aria-label="Full size table 2"
                    ><span>Full size table</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xmlns:xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use>
                      </svg></a>
                </div>
              </figure>
            </div>

            <p>
              克隆型分布分析包括克隆型数目统计、丰度（abundance）分析、CDR3序列长度分析等，旨在获得样本中不同克隆型的大致分布情况。
              本分析和下面的大部分分析内容采用immunarch
              <sup>
                <a data-track="click" data-track-action="reference anchor" 
                data-track-label="link" data-test="citation-ref" aria-label="Reference 1" 
                title="ImmunoMind Team. (2019). immunarch: An R Package for Painless Bioinformatics Analysis of T-Cell and B-Cell Immune Repertoires. Zenodo. http://doi.org/10.5281/zenodo.3367200" 
                href="./report.html#ref-CR1" id="ref-link-section-d38771043e817">1</a>
              </sup>
              进行分析。
            </p>
            <p>
              克隆型数目统计可以获得样本中总克隆型数目和样本特异性的克隆型数目；丰度分析探索了克隆型的富集情况，比如最富集的10%克隆型的细胞占有的比例；
              CDR3序列长度分析统计了样本中CDR3序列长度的分布，可以辅助判断样本的差异。
            </p>
            <div class="
                c-article-section__figure
                js-c-reading-companion-figures-item
              " data-test="figure" data-container-section="figure" id="figure-1" data-title="克隆型分布" data-gtm-vis-first-on-screen-10482319_399="746637" data-gtm-vis-total-visible-time-10482319_399="10000" data-gtm-vis-recent-on-screen-10482319_399="4647217" data-gtm-vis-has-fired-10482319_399="1">
              <figure>
                <figcaption>
                  <b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: 克隆型分布</b>
                </figcaption>
                <div class="c-article-section__figure-content">
                  {% for immune_type in summary.immune_types %}
                  <p>{{immune_type}}</p>
                  <div class="c-article-section__figure-item">
                    <a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="./full_size_page/Fig1.html" rel="nofollow"><picture><source type="image/webp" srcset="
                      show_img/{{ immune_type }}_fig1.png 1x
                          ">
                        <img aria-describedby="Fig0" src="show_img/{{ immune_type }}_fig1.png" alt="figure0" loading="lazy" width="685" height="519"></picture></a>
                  </div>
                  {% endfor %}
                  <div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc">
                    <p>
                      <b>a</b> 各样本中克隆型总数目。 
                      <b>b</b> 各样本中克隆型丰度变化，横轴表示克隆型具有的细胞数目，纵轴表示克隆型的数目。
                      <b>c</b> 克隆型频率在对应分段的克隆型的累计占比，如淡黄色表示频率小于10<sup>-4</sup>的的克隆型的累计百分比。
                      <b>d</b> 对应克隆细胞数目占样本细胞的比例，颜色表示克隆细胞数目。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="./full_size_page/Fig1.html" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon">
                      <use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a>
                </div>
              </figure>
            </div>
          </div>
        </div>
      </section>
      {% endif %}

      {% if check_sec.sample_merge == 1 %}
      <section data-title="共享克隆型">
        <div class="c-article-section" id="Sec11-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="Sec11"
          >
          共享克隆型
          </h2>
          <div class="c-article-section__content" id="Sec11-content">
            <p>
              样本中的共享克隆型（Repertoire overlap）揭示了样本的相似性和可能存在的细胞迁移路径，共享克隆型越多，样本相似性越高。
            </p>
            <div
              class="
                c-article-section__figure
                js-c-reading-companion-figures-item
              "
              data-test="figure"
              data-container-section="figure"
              id="figure-2"
              data-title="Overview of CellChat."
            >
              <figure>
                <figcaption>
                  <b
                    id="Fig2"
                    class="c-article-section__figure-caption"
                    data-test="figure-caption-text"
                    >Fig. 2: 共享克隆型</b
                  >
                </figcaption>
                <div class="c-article-section__figure-content">
                  {% for immune_type in summary.immune_types %}
                  <p>{{immune_type}}</p>
                  <div class="c-article-section__figure-item">
                    <a class="c-article-section__figure-link" data-test="img-link" data-track="click" 
                    data-track-label="image" data-track-action="view figure" href="./full_size_page/Fig2.html" rel="nofollow">
                      <picture><source type="image/webp" srcset="
                      show_img/{{ immune_type }}_fig2.png 1x
                          ">
                        <img aria-describedby="Fig2" src="show_img/{{ immune_type }}_fig2.png" alt="figure0" loading="lazy" width="685" height="519"></picture></a>
                  </div>
                  {% endfor %}
                  <div
                    class="c-article-section__figure-description"
                    data-test="bottom-caption"
                    id="figure-2-desc"
                  >
                    <!-- <p>
                      文库的整合优先选用Seurat中的“SCTransform”
                      <sup>
                        <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Hafemeister C, Satija R. Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression[J]. Genome biology, 2019, 20(1): 1-15." href="./report.html#ref-CR1" id="ref-link-section-d38771043e817">1</a>
                      </sup>
                      。“SCTransform”不仅批次效应矫正能力强，而且能够更好保留真实存在的生物学差异。
                      <b>a</b> 不同批次整合后的细胞在UMAP坐标上的情况，不同颜色表示不同批次，通过这个图我们可以观察不同批次数据的整合后的情况。许多情况下，我们期望不同batch细胞弥散分布，没有明显的聚集。但是，即使某个batch的细胞有聚集，也不一定是整合效果不好。可能不同batch样本的差异巨大，存在生物学上特有的细胞类群。但生物学重复如果出现batch特异，则需要额外注意。  
                      <b>b</b> 不同样本在UMAP坐标上的展示情况，不同颜色表示不同样本。在聚类之后，我们可以更为定量地展示是否某个cluster在不同batch（<b>c</b>）或者样本间（<b>d</b>）的差异。类似<b>a</b>和<b>b</b>的判断方法，<b>c</b>和<b>d</b>可以辅助质控评判和挑选condition特异的细胞类群。

                    </p> -->
                    <p>
                      <b>a</b> 不同样本间overlap的克隆型数目。 
                      <b>b</b> Morisita指数，评估样本重叠度，指数越接近1代表两个样本重叠度越高。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="article-link"
                    data-track="click"
                    data-track-label="button"
                    data-track-action="view figure"
                    href="./full_size_page/Fig2.html"
                    data-track-dest="link:Figure1 Full size image"
                    aria-label="Full size image figure 1"
                    rel="nofollow"
                    ><span>Full size image</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use></svg></a>
                </div>
              </figure>
            </div>
          </div>
        </div>
      </section>
      {% endif %}

      {% if check_sec.cell_cluster_marker == 1 %}
      <section data-title="克隆多样性">
        <div class="c-article-section" id="Sec12-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="Sec12"
          >
          克隆多样性
          </h2>
          <div class="c-article-section__content" id="Sec12-content">
            <!-- <h3 class="c-article__sub-heading" id="Sec13">
              Database construction for ligand-receptor interactions
            </h3> -->
            <p>
              克隆型多样性（Diversity）和机体的状态相关，在一些病理状态下，可能会出现T细胞克隆性增殖的现象，使克隆型多样性下降。
            </p>

            <div
              class="
                c-article-section__figure
                js-c-reading-companion-figures-item
              "
              data-test="figure"
              data-container-section="figure"
              id="figure-3"
              data-title="Overview of CellChat."
            >
              <figure>
                <figcaption>
                  <b
                    id="Fig3"
                    class="c-article-section__figure-caption"
                    data-test="figure-caption-text"
                    >Fig. 3: 克隆型多样性</b
                  >
                </figcaption>
                <div class="c-article-section__figure-content">
                  {% for immune_type in summary.immune_types %}
                  <p>{{immune_type}}</p>
                  <div class="c-article-section__figure-item">
                    <a class="c-article-section__figure-link" data-test="img-link" data-track="click" 
                    data-track-label="image" data-track-action="view figure" href="./full_size_page/Fig3.html" rel="nofollow">
                      <picture><source type="image/webp" srcset="
                      show_img/{{ immune_type }}_fig3.png 1x
                          ">
                        <img aria-describedby="Fig3" src="show_img/{{ immune_type }}_fig3.png" alt="figure0" loading="lazy" width="685" height="519"></picture></a>
                  </div>
                  {% endfor %}
                  <div
                    class="c-article-section__figure-description"
                    data-test="bottom-caption"
                    id="figure-5-desc"
                  >
                    <p>
                      <b>a</b> chao1指数于1984年由Chao提出，最早用于度量物种丰富度的指标，它和丰度、均匀度无关，但是它对稀有的物种很敏感，chao1指数越大，表示丰富性越好；  
                      <b>b</b> D50统计样本中累计克隆型频率分布达到50%时的克隆型数目，D50越大，表示克隆型分布越均匀；
                      <b>c</b> gini指数是常用于评估收入分配公平的指标，该指针取值在0~1之间，基尼指数越小，表示收入越平均，这里用于评估克隆型丰富度，gini指数越低，丰富度越好；
                      <b>d</b> 稀释性曲线是一种通过抽样评估物种丰富度的方法。对细胞进行多次抽样，利用样本中的克隆型数目构建曲线。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="article-link"
                    data-track="click"
                    data-track-label="button"
                    data-track-action="view figure"
                    href="./full_size_page/Fig3.html"
                    data-track-dest="link:Figure1 Full size image"
                    aria-label="Full size image figure 1"
                    rel="nofollow"
                    ><span>Full size image</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use></svg></a>
                </div>
              </figure>
            </div>
          </div>
        </div>
      </section>
      {% endif %}

      {% if check_sec.go_kegg == 1 %}
      <section data-title="V(D)J基因频率">
        <div class="c-article-section" id="code-availability-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="富集分析"
          >
          V(D)J基因频率
          </h2>
          <div
            class="c-article-section__content"
            id="code-availability-content"
          >
          <p>
            样本V(D)J基因频率（Geneusage）反应了T/BCR克隆型的总体特征，对样本V(D)J基因可以获得出现频率最高的V、D、J基因，以及出现频率最高的V-J基因对，这些结果有助于对克隆型功能的理解。
            这里仅展示单个样本的结果。
          </p>
          <div
              class="
                c-article-section__figure
                js-c-reading-companion-figures-item
              "
              data-test="figure"
              data-container-section="figure"
              id="figure-4"
              data-title="Overview of CellChat."
            >
              <figure>
                <figcaption>
                  <b
                    id="Fig4"
                    class="c-article-section__figure-caption"
                    data-test="figure-caption-text"
                    >Fig. 4: V(D)J基因频率</b
                  >
                </figcaption>
                <div class="c-article-section__figure-content">
                  {% for immune_type in summary.immune_types %}
                  <p>{{immune_type}}</p>
                  <div class="c-article-section__figure-item">
                    <a class="c-article-section__figure-link" data-test="img-link" data-track="click" 
                    data-track-label="image" data-track-action="view figure" href="./full_size_page/Fig4.html" rel="nofollow">
                      <picture><source type="image/webp" srcset="
                      show_img/{{ immune_type }}_fig4.png 1x
                          ">
                        <img aria-describedby="Fig4" src="show_img/{{ immune_type }}_fig4.png" alt="figure0" loading="lazy" width="685" height="519"></picture></a>
                  </div>
                  {% endfor %}
                  <div
                    class="c-article-section__figure-description"
                    data-test="bottom-caption"
                    id="figure-4-desc"
                  >
                    <p>
                      <b>a，b</b> 分别用柱状图和极坐标图展示top20的V基因的出现频率。
                      <b>c</b> V-J基因对的出现频率。
                      <b>d</b> 基因V(D)J基因频率J-S散度。J-S散度用于评估两样本中基因的重叠度，如果值越大，则重叠度越低。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="article-link"
                    data-track="click"
                    data-track-label="button"
                    data-track-action="view figure"
                    href="./full_size_page/Fig4.html"
                    data-track-dest="link:Figure1 Full size image"
                    aria-label="Full size image figure 1"
                    rel="nofollow"
                    ><span>Full size image</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use></svg></a>
                </div>
              </figure>
            </div>
          </div>
        </div>
      </section>
      {% endif %}

      {% if check_sec.pseudotime == 1 %}
      <section data-title="克隆型追踪">
        <div class="c-article-section" id="article-comments-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="克隆型追踪"
          >
          克隆型追踪
          </h2>
          <div
            class="c-article-section__content"
            id="article-comments-content"
          >
            <p>
              克隆型追踪（Clonetype tracking）可以显示感兴趣的克隆型在不同的样本或者类群中的分布情况。这里显示的的是这里显示的是前两个样本top10的克隆型在不同样本中的分布。
            </p>
            <div
              class="
                c-article-section__figure
                js-c-reading-companion-figures-item
              "
              data-test="figure"
              data-container-section="figure"
              id="figure-5"
              data-title="Overview of CellChat."
            >
              <figure>
                <figcaption>
                  <b
                    id="Fig5"
                    class="c-article-section__figure-caption"
                    data-test="figure-caption-text"
                    >Fig. 5: 克隆型追踪</b
                  >
                </figcaption>
                <div class="c-article-section__figure-content">
                  {% for immune_type in summary.immune_types %}
                  <p>{{immune_type}}</p>
                  <div class="c-article-section__figure-item">
                    <a class="c-article-section__figure-link" data-test="img-link" data-track="click" 
                    data-track-label="image" data-track-action="view figure" href="./full_size_page/Fig5.html" rel="nofollow">
                      <picture><source type="image/webp" srcset="
                      show_img/{{ immune_type }}_fig5.png 1x
                          ">
                        <img aria-describedby="Fig5" src="show_img/{{ immune_type }}_fig5.png" alt="figure0" loading="lazy" width="685" height="519"></picture></a>
                  </div>
                  {% endfor %}
                  <div
                    class="c-article-section__figure-description"
                    data-test="bottom-caption"
                    id="figure-5-desc"
                  >
                    <p>
                      <b>a，b</b> 分别展示在第一个和第二个样本中，基于CDR3氨基酸序列丰度前10的克隆型在不同样本间分布情况。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="article-link"
                    data-track="click"
                    data-track-label="button"
                    data-track-action="view figure"
                    href="./full_size_page/Fig5.html"
                    data-track-dest="link:Figure1 Full size image"
                    aria-label="Full size image figure 1"
                    rel="nofollow"
                    ><span>Full size image</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use></svg></a>
                </div>
              </figure>
            </div>
          </div>
        </div>
      </section>
      {% endif %}

      {% if check_sec.cell_type == 1 %}
      <section data-title="Motif分析">
        <div class="c-article-section" id="article-comments-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="Motif分析"
          >
          Motif分析
          </h2>
          <div
            class="c-article-section__content"
            id="article-comments-content"
          >
            <p>
              通过对CDR3序列的motif分析可以获得样本中最富集的Kmer，用于判断T/BCR对抗原识别的特异性，具有相似Kmer的T/BCR具有相近的结构和亲和性，因而对相似抗原具有识别能力。
            </p>
            <div
              class="
                c-article-section__figure
                js-c-reading-companion-figures-item
              "
              data-test="figure"
              data-container-section="figure"
              id="figure-6"
              data-title="Overview of CellChat."
            >
              <figure>
                <figcaption>
                  <b
                    id="Fig6"
                    class="c-article-section__figure-caption"
                    data-test="figure-caption-text"
                    >Fig. 6: Motif序列分析</b
                  >
                </figcaption>
                <div class="c-article-section__figure-content">
                  {% for immune_type in summary.immune_types %}
                  <p>{{immune_type}}</p>
                  <div class="c-article-section__figure-item">
                    <a class="c-article-section__figure-link" data-test="img-link" data-track="click" 
                    data-track-label="image" data-track-action="view figure" href="./full_size_page/Fig6.html" rel="nofollow">
                      <picture><source type="image/webp" srcset="
                      show_img/{{ immune_type }}_fig6.png 1x
                          ">
                        <img aria-describedby="Fig6" src="show_img/{{ immune_type }}_fig6.png" alt="figure6" loading="lazy" width="685" height="519"></picture></a>
                  </div>
                  {% endfor %}
                  <div
                    class="c-article-section__figure-description"
                    data-test="bottom-caption"
                    id="figure-6-desc"
                  >
                    <p>
                      <b>a</b> 计算样本中长度为5个氨基酸的Kmer分布，展示的是丰度前10的CDR3氨基酸kmer序列在不同样本中的分布情况。
                      <b>b</b> 长度为5的kmer的motif图。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="article-link"
                    data-track="click"
                    data-track-label="button"
                    data-track-action="view figure"
                    href="./full_size_page/Fig6.html"
                    data-track-dest="link:Figure1 Full size image"
                    aria-label="Full size image figure 1"
                    rel="nofollow"
                    ><span>Full size image</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use></svg></a>
                </div>
              </figure>
            </div>
          </div>
        </div>
      </section>

      <section data-title="UMAP">
        <div class="c-article-section" id="article-comments-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="UMAP"
          >
          UMAP
          </h2>
          <div
            class="c-article-section__content"
            id="article-comments-content"
          >
            <p>
              在UMAP上展示克隆型丰度有助于理解类群中克隆型的分布，通过将统计得到的克隆型细胞数目与转录组数据整合，利用Seurat软件绘图。
            </p>
            <div
              class="
                c-article-section__figure
                js-c-reading-companion-figures-item
              "
              data-test="figure"
              data-container-section="figure"
              id="figure-7"
              data-title="Overview of CellChat."
            >
              <figure>
                <figcaption>
                  <b
                    id="Fig7"
                    class="c-article-section__figure-caption"
                    data-test="figure-caption-text"
                    >Fig. 7: 克隆型大小在UMAP上的分布</b
                  >
                </figcaption>
                <div class="c-article-section__figure-content">
                  {% for immune_type in summary.immune_types %}
                  <p>{{immune_type}}</p>
                  <div class="c-article-section__figure-item">
                    <a class="c-article-section__figure-link" data-test="img-link" data-track="click" 
                    data-track-label="image" data-track-action="view figure" href="./full_size_page/Fig7.html" rel="nofollow">
                      <picture><source type="image/webp" srcset="
                      show_img/{{ immune_type }}_fig7.png 1x
                          ">
                        <img aria-describedby="Fig7" src="show_img/{{ immune_type }}_fig7.png" alt="figure7" loading="lazy" width="685" height="519"></picture></a>
                  </div>
                  {% endfor %}
                  <div
                    class="c-article-section__figure-description"
                    data-test="bottom-caption"
                    id="figure-7-desc"
                  >
                    <p>
                      <b>a</b> 克隆型大小图。对克隆型大小进行分类，细胞没有检测出克隆型的细胞显示为NA。在结果文件中也显示克隆型大小绝对值在UMAP上的分布。
                      <b>b</b> 克隆型频率图。对克隆型按照频率进行分类，克隆型占细胞总体的比例对应分别为：Hyperexpanded (0.01~1)、Large（0.001~0.01）、Medium（1e-04~0.001）、small（< 1e-04)。
                    </p>
                  </div>
                </div>
                <div class="u-text-right u-hide-print">
                  <a
                    class="c-article__pill-button"
                    data-test="article-link"
                    data-track="click"
                    data-track-label="button"
                    data-track-action="view figure"
                    href="./full_size_page/Fig7.html"
                    data-track-dest="link:Figure7 Full size image"
                    aria-label="Full size image figure 7"
                    rel="nofollow"
                    ><span>Full size image</span
                    ><svg
                      width="16"
                      height="16"
                      focusable="false"
                      role="img"
                      aria-hidden="true"
                      class="u-icon"
                    >
                      <use
                        xlink="http://www.w3.org/1999/xlink"
                        xlink:href="#global-icon-chevron-right"
                      ></use></svg></a>
                </div>
              </figure>
            </div>
          </div>
        </div>
      </section>
      {% endif %}

      <!-- <section data-title="模板">
        <div class="c-article-section" id="article-comments-section">
          <h2
            class="
              c-article-section__title
              js-section-title js-c-reading-companion-sections-item
            "
            id="模板"
          >
            模板
          </h2>
          <div
            class="c-article-section__content"
            id="article-comments-content"
          >
            <p>
              内容
            </p>
          </div>
        </div>
      </section> -->



      <div id="MagazineFulltextArticleBodySuffix">
        <section aria-labelledby="Bib1" data-title="References">
          <div class="c-article-section" id="Bib1-section">
            <h2
              class="
                c-article-section__title
                js-section-title js-c-reading-companion-sections-item
              "
              id="Bib1"
            >
              References
            </h2>
            <div class="c-article-section__content" id="Bib1-content">
              <div data-container-section="references">
                <ol class="c-article-references">
                  <li
                    class="
                      c-article-references__item
                      js-c-reading-companion-references-item
                    "
                  >
                    <span class="c-article-references__counter">1.</span>
                    <p class="c-article-references__text" id="ref-CR1">
                      Chao A. Nonparametric estimation of the number of classes in a population[J]. Scandinavian Journal of statistics, 1984: 265-270.
                    </p>
                  </li>
                  <li
                    class="
                      c-article-references__item
                      js-c-reading-companion-references-item
                    "
                  >
                    <span class="c-article-references__counter">2.</span>
                    <p class="c-article-references__text" id="ref-CR2">
                      Gini C. On the measure of concentration with special reference to income and statistics[J]. Colorado College Publication, General Series, 1936, 208(1): 73-79.
                    </p>
                  </li>
                  <li
                    class="
                      c-article-references__item
                      js-c-reading-companion-references-item
                    "
                  >
                    <span class="c-article-references__counter">3.</span>
                    <p class="c-article-references__text" id="ref-CR3">
                      ImmunoMind Team. (2019). immunarch: An R Package for Painless Bioinformatics Analysis of T-Cell and B-Cell Immune Repertoires. Zenodo. 
                    </p>
                  </li>
                  <li
                    class="
                      c-article-references__item
                      js-c-reading-companion-references-item
                    "
                  >
                    <span class="c-article-references__counter">4.</span>
                    <p class="c-article-references__text" id="ref-CR4">
                      Hao Y, Hao S, Andersen-Nissen E, et al. Integrated analysis of multimodal single-cell data[J]. Cell, 2021, 184(13): 3573-3587. e29.
                    </p>
                  </li>
                  
                </ol>

              </div>
            </div>
          </div>
        </section>
      </div>


      <!-- <span data-recommended="jobs"></span> -->
    </div>
  </article>
</main>

<aside
  class="c-article-extras u-hide-print"
  aria-label="Article navigation"
  data-component-reading-companion=""
  data-container-type="reading-companion"
  data-track-component="reading companion"
>
  <div class="c-pdf-download u-clear-both">
    <a
      href="./result_file/report_cn.pdf"
      class="c-pdf-download__link"
      data-article-pdf="true"
      data-readcube-pdf-url="true"
      data-test="download-pdf"
      data-draft-ignore="true"
      data-track="click"
      data-track-action="download pdf"
      data-track-label="link"
      data-track-external=""
    >
      <span>Download PDF</span>
      <svg
        aria-hidden="true"
        focusable="false"
        width="16"
        height="16"
        class="u-icon"
      >
        <use xlink:href="#global-icon-download"></use>
      </svg>
    </a>
  </div>

  <div class="c-reading-companion">
    <div
      class="c-reading-companion__sticky"
      data-component="reading-companion-sticky"
      data-test="reading-companion-sticky"
      style="top: 40px"
    >
      <div
        class="
          c-reading-companion__panel
          c-reading-companion__sections
          c-reading-companion__panel--active
        "
        id="tabpanel-sections"
        aria-labelledby="tab-sections"
      >
        <div class="u-mt-16" data-component-mpu="">
        </div>
      </div>
      <div
        class="
          c-reading-companion__panel
          c-reading-companion__figures
          c-reading-companion__panel--full-width
        "
        id="tabpanel-figures"
        aria-labelledby="tab-figures"
      >
      </div>
      <div
        class="
          c-reading-companion__panel
          c-reading-companion__references
          c-reading-companion__panel--full-width
        "
        id="tabpanel-references"
        aria-labelledby="tab-references"
      >
     </div>
    </div>
  </div>
</aside>
